server/src/main/java/io/crate/expression/symbol/OuterColumn.java

```diff
@@ -70,11 +69,6 @@ public boolean isDeterministic() {
         return symbol.isDeterministic();
     }
 
-    @Override
-    public boolean any(Predicate<? super Symbol> predicate) {
```

Comment:
See docs update in `Symbol`. This matches the `DefaultTraversalSymbolVisitor` behavior and otherwise some correlated join tests broke because the relation name collection would include relation names from the parent.